lib/vmpooler/api.rb

```diff
@@ -205,6 +207,7 @@ def mean(list)
           end
 
           if result[:status][:empty]
+            status 503
```

Comment:
Returning a `503` says that the request to this URL had an issue and could not be processed. I think you are saying that the pooler itself is in a bad state - not the HTTP request itself - and I think the `result[:status][:ok] = false` relays that information.

I would like this to return a `200 OK` regardless of the pooler's pool status.